CHANGELOG.md

```diff
@@ -11,12 +11,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a new image viewer that appears when you tap an image.
 - Included the npub in the properties list sent to analytics.
 - Removed the like and repost counts from the Main and Profile feeds.
-- Replaced hard-coded color values.
+- Removed wss:// from relay adresses in lists and removed the need to prepend relay addresses with wss://.
```

Comment:
```suggestion
- Removed wss:// from relay addresses in lists and removed the need to prepend relay addresses with wss://.
```

Reply:
Just to be clear:  no rgb comment at all, and a correctly spelled comment about wss:// in the unreleased section.

Reply:
@lingoslinger sorry I missed your comment until now. Absolutely right on the second part, but the first part should be: "no duplicate rgb comment" since we had it both in 0.1.25 _and_ in Unreleased.